### ISSUE TITLE
victoria-metrics-k8s-stack not install depended charts

### ISSUE THREAD

**Issue by kevinvirs** [VictoriaMetrics/helm-charts#135]
NAME                            CHART VERSION   APP VERSION     DESCRIPTION
vm/victoria-metrics-k8s-stack   0.2.1           1.16.0          Kubernetes monitoring on VictoriaMetrics stack.

with this values.yaml (cropped)
helm install vm vm/victoria-metrics-k8s-stack -f values.yaml
```
vmsingle:
  enabled: true
vmalert:
  enabled: true
vmagent:
  enabled: true
prometheus-node-exporter:
  enabled: true
kube-state-metrics:
  enabled: true
kubelet:
  enabled: true
kubeApiServer:
  enabled: true
kubeControllerManager:
  enabled: true
coreDns:
  enabled: true
kubeEtcd:
  enabled: true
kubeScheduler:
  enabled: true
```
install only agent/alert/single
```
vmagent-vm-victoria-metrics-k8s-stack-8955b5cff-nrsdf    2/2     Running   0          54s
vmalert-vm-victoria-metrics-k8s-stack-5cdf47d89b-ssj8v   2/2     Running   0          54s
vmsingle-vm-victoria-metrics-k8s-stack-dc675fbc4-nv8ff   1/1     Running   0          55s
```
not created cluster roles, kube-state, node-exporter, operator resources
helm template in attachment
[render-template.txt](https://github.com/VictoriaMetrics/helm-charts/files/6419631/render-template.txt)

**Comment by kevinvirs** [comment]
Created pull-request https://github.com/VictoriaMetrics/helm-charts/pull/136
now working correct;y

**Comment by f41gh7** [comment]
merged and bumped new release `0.2.2`
